_events/2017-05-24-swampup.md

```diff
@@ -0,0 +1,32 @@
+---
+# Event title.
+title: JFrog swampUp
+
+# Event Location
+location: Napa Valley, CA
```

Comment:
CA stands for canada too. How about "Napa Valley, CA, USA" or "Napa Valley, California"